app/src/main/java/com/tavarus/artabletop/Controllers/BoardController.kt

```diff
@@ -0,0 +1,27 @@
+package com.tavarus.artabletop.Controllers
+
+import android.content.Context
+import com.google.ar.sceneform.Node
+import com.tavarus.artabletop.Objects.Board
+import com.tavarus.artabletop.POJO.BoardListPOJO
+
+class BoardController(){
+    companion object {
+        var INSTANCE: BoardController = BoardController()
+    }
+
+    var currentBoard: Board? = null
+
+    fun initialize(list: BoardListPOJO, context: Context) {
+        currentBoard = Board(0.05f, list.boards[0], context)
```

Comment:
It also isn't clear what you're using from `list` in the second argument. An extension function like `BoardListPOJO.selectMainBoard()`, or whatever the purpose is, would be clearer.